app/assets/stylesheets/content/_simple_filters.lsg

```diff
@@ -8,6 +8,7 @@ By default, simple filters have a two columned layout.
 @full-width
 
 <fieldset class="simple-filters--container">
+  <a title="Close filter" class="simple-filters--close icon-context icon-close" onClick="hideFilter(jQuery(this.parentElement))"></a>
```

Comment:
`onClick="hideFilter(jQuery(this.parentElement))"` is causing an exception deep down in the living styleguide since minsyntax tries to highlight an attribute that has been cleaned. I'm removing this attribute.